modules/social_features/social_tagging/social_tagging.module

```diff
@@ -522,6 +523,18 @@ function social_tagging_form_views_exposed_form_alter(&$form, FormStateInterface
         $form[$label]['#type'] = 'select2';
         $form[$label]['#size'] = NULL;
 
+        if (isset($form[$label]['#context']['#plugin_type']) && $form[$label]['#context']['#plugin_type'] == 'bef') {
```

Comment:
```suggestion
        if (isset($form[$label]['#context']['#plugin_type']) && $form[$label]['#context']['#plugin_type'] === 'bef') {
```

Reply:
Hi, @ksis Could you make this change and we will be free to merge PR. Thanks!

Reply:
Hi @Ressinel,
I've applied the changes.